apps/i18n/netsim/en_ploc.json

```diff
@@ -0,0 +1,3 @@
+{
+  "yes": "Yes"
```

Comment:
You're correct, pseudoloc decorates the strings with !! so that it's easy to identify if a string has been localized or not.